src/Chirp.Razor/Pages/Public.cshtml

```diff
@@ -27,6 +28,7 @@
     }
     else
     {
-        <em>There are no cheeps so far.</em>
+    <em>There are no cheeps so far.</em>
     }
-</div>
+    <small>page @Model.PageNumber</small>
```

Comment:
add buttons here too

Reply:
Probably would be a good idea make a function or something for this

Reply:
Thanks for very good feedback!
64381b11297597d1d2ec9577780e7b80e1754170 hopefully fixes 

> add buttons here too

What should I make a function for exactly?

Reply:
I only meant to make it so we can reuse the code for buttons but this is good 👍